.github/workflows/integration-test-upgrade-suite.yaml

```diff
@@ -49,7 +49,7 @@ jobs:
     - name: TestCephUpgradeSuite
       run: |
        export DEVICE_FILTER=$(lsblk|awk '/14G/ {print $1}'| head -1)
-       go test -v -timeout 1800s -failfast -run CephUpgradeSuite github.com/rook/rook/tests/integration
+       go test -v -timeout 1800s -failfast -run CephUpgradeSuite/TestUpgradeToMaster github.com/rook/rook/tests/integration
```

Comment:
Maybe it would be good to name the tests a little more clearly now? Like `CephUpgradeSuite/TestUpgradeRookToMaster`?